index.js

```diff
@@ -188,6 +206,7 @@ const image = {
 
     // We support the file type
     token.consume(sentToken, matches[1], function (err) {
+        // OK #noKidding: Node-sqlite3 bind the `this`, so ES6 syntax is not possible here
         if (err || this.changes !== 1) {
```

Comment:
`consume` is your own method. You could make it such that it accepts a callback that accepts the result as its second argument. And yes, node-sqlite3 handles this weirdly as is described in their help.

```javascript
consume(token, id, callback) {
  consumeToken.run([token, id], function(err) {
    callback(err, this);
  });
}
```

Now you can use lamdas in any function calling consume. Not sure if it's the way to go, but it moves this nonsense somewhat higher up the chain.